python/pyspark/sql/tests/connect/test_connect_creation.py

```diff
@@ -706,6 +706,28 @@ def test_simple_udt(self):
 
             self.assertEqual(cdf.schema, sdf.schema)
 
+    def test_empty_df_with_nested_array_columns_conversion_to_pandas_df(self):
```

Comment:
Shall we move this to `test_collection.py`?
The test there should be shared with both classic and connect to see the behavior is the same.